### PR TITLE
Show token expiration times

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

**Comment by liggitt** [comment]
@deads2k review

File: pkg/cmd/cli/describe/printer.go [hunk 1]
```diff
@@ -427,7 +428,9 @@ func printOAuthClientAuthorizationList(list *oauthapi.OAuthClientAuthorizationLi
 }
 
 func printOAuthAccessToken(token *oauthapi.OAuthAccessToken, w io.Writer) error {
-	_, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%d\t%s\t%s\n", token.Name, token.UserName, token.ClientName, token.CreationTimestamp, token.ExpiresIn, token.RedirectURI, strings.Join(token.Scopes, ","))
+	created := token.CreationTimestamp
```
Comment (deads2k) [inline pkg/cmd/cli/describe/printer.go]: Storing this and the immediately using the original read weird.  Not that I doubt their equivalence, but you want to make up your mind one way or the other?

**Comment by deads2k** [comment]
I think users will prefer having to perform computation in their head and write awesome bash scripts to compute the date, but if you insist...

Nit, you can decide one way other.  Merge at will.

**Comment by liggitt** [comment]
[merge]

**Pull request merged**